agenthub/monologue_agent/utils/memory.py

```diff
@@ -7,15 +7,15 @@
 from opendevin import config
 from . import json
 
-embedding_strategy = config.get_or_default("LLM_EMBEDDING_MODEL", "local")
+embedding_strategy = config.get("LLM_EMBEDDING_MODEL")
 
 # TODO: More embeddings: https://docs.llamaindex.ai/en/stable/examples/embeddings/OpenAI/
 # There's probably a more programmatic way to do this.
 if embedding_strategy == "llama2":
     from llama_index.embeddings.ollama import OllamaEmbedding
     embed_model = OllamaEmbedding(
         model_name="llama2",
-        base_url=config.get_or_default("LLM_BASE_URL", "http://localhost:8000"),
+        base_url=config.get_or_error("LLM_BASE_URL"),
```

Comment:
This change will influence a little about the code logic. But I think the previous code logic is not good. Should not provide a defaul localhost address for `llama2` model. I think it will be quickly find and fixed in future if someone meet it.